packages/financial-templates-lib/test/clients/OptimisticOracleClient.js

```diff
@@ -16,6 +16,11 @@ const Timer = getTruffleContract("Timer", web3);
 const Store = getTruffleContract("Store", web3);
 const MockOracle = getTruffleContract("MockOracleAncillary", web3);
 
+const objectsInArrayInclude = (superset, subset) => {
```

Comment:
shoudl probably pull this out as a utility if its used a lot

Reply:
The issue is (as stated above) that different tests order their args differently. I can refactor this if you think it's important, was just trying to avoid unnecessary changes.